contracts/anchor/src/test_util.rs

```diff
@@ -0,0 +1,216 @@
+use ark_bn254::Bn254;
+use ark_ff::{BigInteger, PrimeField};
+use arkworks_circuits::setup::{
+    anchor::{
+        setup_leaf_with_privates_raw_x5_4, setup_leaf_x5_4, setup_proof_x5_4, AnchorProverSetup,
+    },
+    common::{AnchorProof, Leaf},
+};
+use arkworks_utils::utils::common::{setup_params_x5_3, setup_params_x5_4, Curve};
+
+// wasm-utils dependencies
```

Comment:
We don't need comments like this. It's clear from the package name that this is what it is.

Reply:
Removed the comment as the review.
https://github.com/webb-tools/protocol-cosmwasm/pull/15/commits/c52ce0975594794a6b90c5869310bb8a306db47e